### PR TITLE
Added the code for Energy Frequency Visualizer

### LINKED ISSUE #81: [feature] Add Frequency vs Energy Visualization in SoundScape
I would like to add Python code that could plot Frequency vs Energy for a particular sound.

### PR DESCRIPTION
**Pull request by Saksh8**
Closes #81

### CODE REVIEW

File: Energy-Frequency-Visualizer.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+def resume():
+    if not _VARS["stream"].is_active():
+        _VARS["stream"].start_stream()
+        _VARS["window"]["Pause"].Update(disabled=False)
+        _VARS["window"]["Resume"].Update(disabled=True)
+
+def save():
+    folder = sg.popup_get_folder('Please select a directory to save the files')
+    if folder:
+        fig.savefig(f'{folder}/output.png')
```
Comment (dinxsh) [inline Energy-Frequency-Visualizer.py:95]: could you add exception handling here

**Review by dinxsh — APPROVED** [review]
looks good to me, should be ready for merge!

File: Energy-Frequency-Visualizer.py (hunk 1 above)
Comment (Soumya-Kushwaha) [reply Energy-Frequency-Visualizer.py:95]: @Saksh8 pls do the needful.

File: Energy-Frequency-Visualizer.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 sg.theme("DarkBlue3")
 
 menu_layout = [
-    ['Run Visualizers', ['Energy-Frequency-Visualizer', 'Waveform', 'Spectogram', 'Intensity-vs-Frequency-and-time']],
+    ['Run Visualizers', ['Energy-Frequency-Visualizer', 'Waveform', 'Spectrogram', 'Intensity-vs-Frequency-and-time']],
```
Comment (dinxsh) [inline Energy-Frequency-Visualizer.py:24]: its spectogram

File: Energy-Frequency-Visualizer.py [hunk 3]
```diff
@@ -141,9 +155,12 @@ def close_current_visualizer():
         stop()
     if event == "Save":
         save()
-    if event in ['Energy-Frequency-Visualizer', 'Waveform', 'Spectogram', 'Intensity-vs-Frequency-and-time']:
+    if event in ['Energy-Frequency-Visualizer', 'Waveform', 'Spectrogram', 'Intensity-vs-Frequency-and-time']:
```
Comment (dinxsh) [inline Energy-Frequency-Visualizer.py:158]: spectogram not spectrogram

**Review by dinxsh — CHANGES_REQUESTED** [review]
minor typo changes, other than that lgtm

**Review by Soumya-Kushwaha — APPROVED** [review]

**Review by Soumya-Kushwaha — APPROVED** [review]

**Comment by Soumya-Kushwaha** [comment]
Please follow the PR template. @Saksh8

**Comment by Saksh8** [comment]
> Please follow the PR template. @Saksh8

I'm unable to follow the PR template on this existing PR.

**Comment by Soumya-Kushwaha** [comment]
> > Please follow the PR template. @Saksh8
> 
> I'm unable to follow the PR template on this existing PR.

Edit the first comment with basic details, plz.
Helps in tracking.

**Review by dinxsh — APPROVED** [review]
LGTM

**Pull request merged**